### ISSUE TITLE
Awesomesauce Support Theme

### ISSUE THREAD

**Issue by gregglind** [mozilla/donate.mozilla.org#732]
@valianttry I did some research

- https://addons.mozilla.org/en-US/developers/theme/submit  makes it really easy.
- https://developer.mozilla.org/en-US/Add-ons/Themes/Background more detailed.
- making it 'secret' might need some amo team help.  

Bascially you need to make a header and footer image (or not), and that's it.  

(I don't know if the images in the them are 'live grabbed' each startup.  if so you can also use that as an activity tracker)

**Comment by valianttry** [comment]
@gregglind this is outstanding. Blake Winton already raised his hand to help or connect me to the right person... we're off to the races. :hand:

**Comment by valianttry** [comment]
cc @bniolet1

**Comment by MattGrimes** [comment]
Blake also mentioned the possibility of a modified version of the Whimsy
addon for donors. Whimsy is amazing:
https://addons.mozilla.org/en-us/firefox/addon/whimsycorn/

**Comment by valianttry** [comment]
Truth Whimsy is amazing. My only hesitation is that we know donors skew older... I'm not suggesting we need Wilford Brimley to be our spokesperson, but maybe Whimsy is for younger folk. I'm open to pushback on this (maybe older folks LOVE dreamy unicorns and i'm out of touch)

**Comment by valianttry** [comment]
Some notes and such:



See: https://developer.mozilla.org/en-US/Add-ons/Themes/Lightweight_themes

Spoke with Gregg Linden and Matt Grimes in User Advocacy -- joined by @bniolet1 and they had the brilliant idea to give away a FREE Firefox theme to donors. What does this accomplish?

**First** -- It gives Firefox users who donate a really awesome "special gift" when they donate that we ONLY promote to donors. This will be delightful and likely improve retention / increase the likelihood of a repeat gift.

**Second and most importantly** -- This allows us to treat the people who have this theme as a "cohort" among Firefox users. This helps us answer important Qs we've been unable to answer previously, such as:

* Do donors have a lower attrition rate than non-donors who use Firefox?
* Do donors keep their Firefox browser updated in higher #s than non donors?
    etc

Tasks:

- [ ] Design a kick-ass theme that donors will love -- e.g. a Firefox logo treatment that is special; a big gold star added to their homepage etc; -- this doesn't necessarily have to be done by MoFo designers. I could reach out to Blake Winton (of whimsycorn fame) for help since our designers are pretty much tapped.
- [ ] Build a mechanism to offer the theme to donors right after they give -- decide where to put it in the flow;
- [ ] Work with User Advocacy on shared metrics / dashboard to track behavior over time for this cohort + compared to other Firefox users.

cc: @adamlofting @edrushka

**Comment by bwinton** [comment]
(As an aside, for the modified version of Whimsy, I was thinking of something almost completely pared-down, and comfortable for the target audience.  Perhaps just replacing the image in about:home with this one:
![kid and dino](https://raw.githubusercontent.com/bwinton/whimsy/master/data/dino%402x.png)
and putting some "Thanks for donating!"-type messages in the urlbar when it's empty…  I would probably stay away from the crazy bookmark-coin sound, and even after curation, the animated gifs might still be a bit much.  I totally don't mind if you want to go with a lightweight theme instead, and I'm sure it'll be amazing no matter what you choose!  :smiley:)

**Comment by edrushka** [comment]
Not sure if it makes sense to incorporate any of this in the theme but here's a taste of the visual direction for the EOY campaign.

**Directory Tile**(final)
![h0fzrdd6tjb6eoeatftg2o-o39hic0gmvpewp01eyx-pl8olz3m5b8v9ru25wcczhunbwg w1251-h460](https://cloud.githubusercontent.com/assets/4515662/11158101/adcb25c2-8a0a-11e5-8f3c-7697b88c0d4c.png)

**Mozilla.org homepage takeover** (*not* final -  this is a preview)
<img width="576" alt="eoy-takeover-preview" src="https://cloud.githubusercontent.com/assets/4515662/11158414/e82b18f6-8a0c-11e5-811e-430cf7a2528c.png">

**Comment by valianttry** [comment]
@bwinton do you think you could create a light theme using dapper fox?

**Comment by bwinton** [comment]
No, my drawing skills are really not that good.  :grinning:
Perhaps @shorlander could whip something out, though…

**Comment by valianttry** [comment]
ha! @bwinton the design is done, just need to take the assets to make a theme that is fun for the whole family! :balloon: We would absolutely LOVE help with this, our designers are really at capacity at the moment. This would be a big help. If @shorlander can help, awesome. Thanks so much!

**Comment by edrushka** [comment]
Here are all the Dapper Fox assets in an AI file: https://www.dropbox.com/s/5g8778rzbfoxb6b/Fundrasing-DAPPER-FOX.ai?dl=0

**Comment by edrushka** [comment]
Also, to clarify, Dapper Fox is what the fundraising team has named the little fox wearing a hat in the graphics above :)

**Comment by shorlander** [comment]
![dapper-fox](https://cloud.githubusercontent.com/assets/684427/11385468/23905630-92e7-11e5-804d-188e6c2d9d49.png)
Two options. Mostly differing positioning of Dapper Fox.

**Comment by valianttry** [comment]
@shorlander this is amazing.

**Comment by valianttry** [comment]
It is perfect in every way. Q: Would it be cheezie / unseemly to have Dapper saying something or acknowledge the donor somehow? E.g. "Thanks for your gift!" or "Your donation keeps Mozilla awesome" or ... something less lame than what I just suggested....

**Comment by edrushka** [comment]
These are super! I think I prefer the first one (less hat) but I defer to the designers! Please let me know when it's ready to share so we can put the link on social.

**Comment by cassiemc** [comment]
@edrushka I like the 'less hat' version too, less tension between the face and the url bar.Thanks @shorlander!

**Comment by shorlander** [comment]
Is there an official AMO account we are submitting the theme under?

**Comment by jbuck** [comment]
@shorlander I don't think we have an official AMO account. Is there one used for other themes/addons that Mozilla puts out?

**Comment by shorlander** [comment]
@jbuck I don't think we have an offical Mozilla user on AMO. That I know of anyway.
